src/main/java/com/elixr/poc/rest/response/GetAllResponse.java

```diff
@@ -7,7 +7,7 @@
 import java.util.List;
 
 /**
- * Sends successful response to the user while retrieval operation
+ * Sends successful response to the user while retrieval operation.
  */
 @Data
 public class GetAllResponse extends AppResponse {
```

Comment:
```suggestion
public class GetAllUsersResponse extends AppResponse {
```

Reply:
As This class is being used by people who are working on purchase so I named it as GetAllRespnse